PANDORA/PMHC/Model.py

```diff
@@ -10,8 +10,8 @@
 
 class Model:
 
-    def __init__(self, target, model_path='', output_dir = PANDORA.PANDORA_data, pdb=False, molpdf=0, dope=0):
-        '''__init__(self, target, model_path='', output_dir = PANDORA.PANDORA_data, pdb=False, molpdf=0, dope=0)
+    def __init__(self, target, model_path='', output_dir = os.getcwd(), pdb=False, molpdf=0, dope=0):
```

Comment:
Please remove the "os.getcwd()" from the function definition. Add an if statement in the beginning of the function instead to assign the output dir if no output dir is provided and specify it in the docstring.